Makefile

```diff
@@ -177,8 +177,10 @@ web-deploy:
 
 # Generates the config file documentation.
 doc: clean-doc
```

Comment:
The `doc` task should be .PHONY as it does not produce a file named doc.

Reply:
Good point, thanks!